platforms/core-runtime/versioned-cache/src/main/java/org/gradle/internal/versionedcache/UnusedVersionsCacheCleanup.java

```diff
@@ -56,7 +59,7 @@ public static UnusedVersionsCacheCleanup create(String cacheName, CacheVersionMa
     private UnusedVersionsCacheCleanup(final Pattern cacheNamePattern, CacheVersionMapping cacheVersionMapping, UsedGradleVersions usedGradleVersions) {
         super((baseDir, filter) -> {
             FileFilter combinedFilter = FileFilterUtils.and(directoryFileFilter(), new RegexFileFilter(cacheNamePattern), asFileFilter(filter),
-                asFileFilter(new NonReservedFileFilter(singleton(baseDir))));
+                FileFilterUtils.asFileFilter(new NonReservedFileFilter(singleton(baseDir))));
```

Comment:
❓ Why qualify this call but not the one above?

Personally I'd keep both unqualified and rename the method if you think it doesn't provide enough clarity.

Reply:
> ❓ Why qualify this call but not the one above?
> 
> Personally I'd keep both unqualified and rename the method if you think it doesn't provide enough clarity.

I've reverted this one. Likely a quirk of the automated refactoring in IDEA.